lifting_veil/agents/dqn_agent_new.py

```diff
@@ -299,6 +316,9 @@ def _train_step(self):
             if self.training_steps % self.update_period == 0:
                 self._sample_from_replay_buffer()
 
+                states = self._preprocess_fn(self.replay_elements['state'])
```

Comment:
you need to pass an rng here:
https://github.com/google/dopamine/blob/master/dopamine/labs/atari_100k/atari_100k_rainbow_agent.py#L104